src/main/resources/interface-specification.yml

```diff
@@ -3112,6 +3193,33 @@ components:
         - DELEGATE
         - SUB_DELEGATE
         - OPERATOR
+    ProducerEService:
+      type: object
+      required:
+        - id
+        - name
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        activeDescriptor:
+          $ref: '#/components/schemas/CompactDescriptor'
```

Comment:
The story specs want an `ActiveDescriptor` object here.
What about extending `CompactDescriptor` with the version field and then removing `ActiveDescriptor`?

Reply:
`CompactDescriptor` already contains the state. I didn't delete `ActiveDescriptor` yet because it's used in other endpoints (and to limit conflicts between our PRs), but if we are ready I can use `CompactDescriptor` everywhere.
Let me know

Reply:
done 👍